src/pages/myNotes.jsx

```diff
@@ -0,0 +1,179 @@
+import React from "react"
+import { useState, useEffect } from "react"
+import StickyNote2Icon from "@mui/icons-material/StickyNote2"
+import InfoIcon from "@mui/icons-material/Info"
+import OfflineShareIcon from "@mui/icons-material/OfflineShare"
+import {
+  BrowserRouter,
+  Switch,
+  Route,
+  NavLink,
+  Redirect,
+} from "react-router-dom"
+import axios from "axios"
+import { createStore } from "redux"
+
+import PATHS from "../../src/config/routes/routes"
+
+import MyNotesContainer from "./myNotesContainer"
+import SharedNotes from "./sharedNotes"
+import AboutApp from "./about"
+import SignUpForm from "./signUp"
+import SignInForm from "./signIn"
+import NotFound from "../components/NotFound"
+import "./App.css"
+import styles from "./styled.js"
+
+const MyNotes = ({ NOTES, SHARED, isAuthorized, submitAutorization }) => {
+  const [myNotes, saveNote] = useState(NOTES)
+
+  useEffect(() => {
+    if (localStorage.myNotes) {
+      saveNote(JSON.parse(localStorage.getItem("myNotes")))
+    } else {
+      console.log(NOTES)
+      saveNote(NOTES)
+    }
+  }, [NOTES])
+
+  if (localStorage.sharedNotes) {
+    SHARED = JSON.parse(localStorage.getItem("sharedNotes"))
+    console.log("called")
+  }
+
+  const [sharedChosenNote, displaySharedNote] = useState({
+    title: "Select note to display",
+  })
+
+  const [myChosenNote, displayMyNote] = useState({
+    title: "Select note to display",
+  })
+  const [num, changeActive] = useState([-1])
+
+  let chosenNote = { num }
+
+  function showChosenNote(id, activeColor, shadow) {
+    const allNotes = document.getElementsByTagName("li")
+    for (let n = 0; n < allNotes.length; n++) {
+      allNotes[n].style.background = "inherit"
+    }
+    const noteInList = allNotes[id]
+    noteInList.style.background = activeColor
+    document.getElementsByClassName("chosenNote")[0].style.boxShadow = shadow

```

Comment:
We can't use inner changing of the styles/content in react. Try to rewrite this logic with any other behaviour